src/device.rs

```diff
@@ -288,18 +294,42 @@ impl DeviceInfo {
             .iter()
             .find(|mgmt_file| mgmt_file.0 == *key)
         {
-            self.meta.map.remove(key);
-            self.get(key)?;
+            self.meta.map.borrow_mut().remove(key);
         }
 
         Ok(())
     }
+
+    pub fn get_numa_node(&self) -> DeviceResult<NumaNode> {
+        if let Some(node) = *self.numa_node.borrow() {
+            return Ok(node);
+        }
+
+        let busname = self.get(sysfs::npu_mgmt::BUSNAME)?;
+        let id = sysfs::pci::numa::read_numa_node(&self.sys_root, &busname)?
+            .parse::<i32>()
+            .unwrap();
+
+        let node = if id == -1 {
+            NumaNode::UnSupported
+        } else if id < 0 {
+            return Err(DeviceError::unexpected_value(format!(
+                "Unexpected numa node id: {}",
+                id
+            )));
+        } else {
+            NumaNode::Id(id as usize)
+        };
```

Comment:
사소한 부분이긴 합니다만, 의미상 중요한 순서대로 나열하는 것은 어떨까요?

```suggestion
        let node = if id >= 0 {
            NumaNode::Id(id as usize)
        } else if id == -1 {
            NumaNode::UnSupported
        } else {
            return Err(DeviceError::unexpected_value(format!(
                "Unexpected numa node id: {}",
                id
            )));
        };
```

Reply:
네, 바로 반영 하겠습니다 감사합니다~